src/www/ui/api/Controllers/JobController.php

```diff
@@ -40,6 +41,19 @@
  */
 class JobController extends RestController
 {
+  public function getAllJobs($request, $response, $args)
+  {
+    $limit = $request->getHeaderLine('limit');
+    $page = $request->getHeaderLine('page');
+    $id = null;
+    if (Auth::isAdmin()) {
+      return $this->getAllResults($id, $response, $limit, $page);
+    } else {
+      $returnVal = new Info(403,'Access Denied', InfoType::ERROR);
+      return $response->withJson($returnVal->getArray(), $returnVal->getCode());
+    }
+  }
+
```

Comment:
Please move the function below the variables. The code should follow a style guide :-)

Reply:
Resloved

Reply:
Resolved